cmd/main.go

```diff
@@ -1 +1,59 @@
 package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/dilyara4949/flight-booking-api/internal/config"
+	"github.com/dilyara4949/flight-booking-api/internal/database/postgres"
+	"github.com/dilyara4949/flight-booking-api/internal/handler"
+	"github.com/dilyara4949/flight-booking-api/internal/repository"
+	"github.com/dilyara4949/flight-booking-api/internal/service"
+	"log/slog"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+func main() {
+	cfg, err := config.NewConfig()
+	if err != nil {
+		slog.Error("error getting config: %w", err)
```

Comment:
are you sure that this will be logged?

Reply:
yes

Reply:
cannot understand why you think so, in case of errors it always logs